systemtests/src/test/java/io/enmasse/systemtest/scale/ScaleTest.java

```diff
@@ -191,5 +258,36 @@ private void setupEnv() throws Exception {
         getResourceManager().createAddressSpace(addressSpace);
         getResourceManager().createOrUpdateUser(addressSpace, userCredentials);
     }
+
+    List<Address> createInitialAddresses(int addresses) throws Exception {
+        int operableAddresses = 0;
+        int iterator = 0;
+
+        while (operableAddresses < addresses) {
+            try {
+                getResourceManager().appendAddresses(false, getTenantAddressBatch(addressSpace).toArray(new Address[0]));
+                if (iterator % 200 == 0) {
```

Comment:
operable addresses are filled only if iterator mod 200 is 0, so if you specify 10 addresses you anyway get 1000 addresses